env-check.sh

```diff
@@ -4,10 +4,20 @@ set -e
 
 mydir=$(cd "$(dirname "$0")"; pwd)
 
+ubuntu_warning()
+{
+cat <<EOF
```

Comment:
Just curious, why the preference of HEREDOC over a one-line `echo -e`?

```bash
echo -e "\n\n\tDisclaimer: this script is tested only on a somewhat recent Ubuntu version.\n\tOn other distributions your mileage may vary.\n\n"
```
Legibility?

Reply:
Because it's two lines long? :-)

Thanks for the review.